src/service/tar_git

```diff
@@ -779,6 +780,12 @@ handle_submodules() {
 }
 
 parse_url () {
+  # Breaks MYURL up into :
+  #   SCHEME://REPO_URL/REPO_PATH/REPO_NAME
```

Comment:
agree tar_git should be re-written in python and use proper libraries to parse this stuff.
This comment says (roughly) what this function creates
I don't know what it should take or what it will do. 
The existing comments do not mention that it supports relative urls - do you know if they're supported?